src/store/reducers/auth-reducer.ts

```diff
@@ -2,13 +2,14 @@ import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
 import { HYDRATE } from 'next-redux-wrapper';
 import { AxiosError } from 'axios';
 import { authApi } from '../../api/authApi';
-import { LoginFormType, RegistrationFormType } from '../../Types/AuthTypes';
+import { LoginFormType, RegistrationFormType } from '../../types/AuthTypes';
 
 export const registration = createAsyncThunk(
   'registration/register',
   async (data: RegistrationFormType, { rejectWithValue }) => {
     try {
       const response = await authApi.registration(data);
+      localStorage.setItem('token', response.data.accessToken);
```

Comment:
Do not forget about `setTokenToStorage` function instead of `localStorage.setItem`
Needs abstraction

Reply:
Yes, i remember about it, i've created this task.